sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/HMICapabilities.java

```diff
@@ -7,8 +7,10 @@
 public class HMICapabilities extends RPCStruct{
     public static final String KEY_NAVIGATION = "navigation";
     public static final String KEY_PHONE_CALL = "phoneCall";
-	
-	 public HMICapabilities() { }
+	public static final String KEY_VIDEO_STREAMING = "videoStreaming";
+	public static final String KEY_AUDIO_STREAMING = "audioStreaming";
```

Comment:
I know the proposal was a little confusing, but the video and audio streaming capabilities aren't included yet